content/updates/2021-04/1-Review-your-proposals-more-efficiently.md

```diff
@@ -0,0 +1,23 @@
+---
+pim_announcement_alt_img: "New proposals screen"
+pim_announcement_audience:
+- EE
+---
+
+# Review your proposals more efficiently
+::: meta-data type="Improvement" features="Productivity, Collaboration" available="late April" in="EE" link-to-doc="../articles/TODO"
+
+We're glad to introduce the revamp of our proposals screen. It's now even more practical and efficient for those of you who, depending on their catalog rights, need to review the enrichment work of teammates. 
+
+Among other improvements, you can notice that:
+- You now have, directly from within the proposal, a preview of your PDFs assets, your YouTube links, and your reference entity records. A great time-saver when reviewing them!
```

Comment:
You forgot to mention the display of images (since the 4.0, we only displayed the asset code. The thumbnail was displayed before the 4.0 (PAM) but in black and white). We can now preview the media but also download them and copy their URL. And we display their label too.
For the records: this is the exact same display as in the PEF (label + thumbnail + link to the ref entity)

Reply:
Looks like I forgot a word. I reformulated to make it better. Is it better this way?